tests/integration/models/application.spec.coffee

```diff
@@ -13,6 +13,79 @@ describe 'Application Model', ->
 
 	givenLoggedInUser()
 
+	describe 'rolling release related methods', ->
```

Comment:
I think this should live inside a `describe 'given ...'` block. We generally use top-level describe blocks to describe the test environment of preexisting data everywhere, and then group by method and functionality like this only within that, where necessary.

If you put this inside `given a single application` block, you'd be able to skip everything before `userId =` since that's already there. I.e. something like:

```coffee
 	describe 'given a single application', ->
 
 		beforeEach ->
 			resin.models.application.create
 				name: 'FooBar'
 				applicationType: 'microservices-starter'
 				deviceType: 'raspberry-pi'
 			.then (application) =>
 				@application = application

# ^^^ this part already exists below
		...

		describe 'given two releases', ->

			beforeEach ->
				userId = @application.user.__id
 				resin.pine.post
					resource: 'release'
					body:
						belongs_to__application: @application.id
						is_created_by__user: userId
						commit: 'old-release-commit'
						status: 'success'
						source: 'cloud'
						composition: {}
						start_timestamp: 1234
 				.then =>
					resin.pine.post
						resource: 'release'
						body:
							belongs_to__application: @application.id
							is_created_by__user: userId
							commit: 'new-release-commit'
							status: 'success'
							source: 'cloud'
							composition: {}
							start_timestamp: 54321

			# Put your tests here
```